packages/web-components/fast-components/src/utilities/components/carousel-test-slide/index.ts

```diff
@@ -0,0 +1,13 @@
+import { customElement } from "@microsoft/fast-element";
+import {
+    CarouselTestSlide,
+    CarouselTestSlideTemplate as template,
+} from "@microsoft/fast-foundation";
+import { CarouselTestSlideStyles as styles } from "./carousel-test-slide.styles";
+
+@customElement({
+    name: "fast-carousel-test-slide",
```

Comment:
Why is it called "test-slide"?

Reply:
Can this live in the fixtures directory then? or maybe in a /test dir in components/carousel?

Reply:
Yes I'll move it to the fixtures directory